packages/test/snapshots/src/test/tsconfig.json

```diff
@@ -8,7 +8,8 @@
             "mocha"
         ],
         "declaration": false,
-        "declarationMap": false
+        "declarationMap": false,
+        "strict": false
```

Comment:
Looks like this is required to run the tests in vscode. If I remove this, the tests don't run with this error:
TSError: ⨯ Unable to compile TypeScript:
packages/test/snapshots/src/replayMultipleFiles.ts(248,27): error TS2454: Variable 'threads' is used before being assigned.